includes/polling/os/unix.inc.php

```diff
@@ -165,9 +165,12 @@
     $aix_descr = explode("\n", $device['sysDescr']);
     # AIX standard snmp deamon
     if ($aix_descr[1]) {
-        $serial = explode("Processor id: ", $aix_descr[1])[1];
+        $serial = substr(snmp_get($device, 'aixSeSerialNumber.0', '-Oqv', 'IBM-AIX-MIB'), -7);
```

Comment:
Could you add a comment here with a sample ```aixSeSerialNumber``` to make it clear why the '-7' is needed ?

Reply:
> Hi @dlesel
> Are you sure your change is not breaking any AIX version out in the wild ?
> If yes, then you'll need to re-run the test-data for aix to make Travis happy.

i am improving. TL is part of AIX Verison: https://www.ibm.com/developerworks/community/blogs/AIXDownUnder/entry/understanding_aix_levels6?lang=en